app/routes/game.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  // TODO: insert yourself into the watchers array unless you're already in one
```

Comment:
You make a good point. I was thinking this approach could be easy now so we'll know for sure who each person participating is. Possibly later we could create anonymous users when you visit without being auth'd, then delete those anonymous users on disconnect?